packages/toolkit/src/view/pipeline/view-pipeline/Head.tsx

```diff
@@ -341,25 +370,12 @@ export const Head = ({
                       router.push("/login");
                     }}
                     className="!normal-case"
-                    variant="secondaryGrey"
+                    variant="secondaryColour"
```

Comment:
@iamnamananand996 Could you help me point out where you get this design?

Reply:
https://www.figma.com/file/eTh2TCwmqN4MxiHJ4vTnPH?type=design&node-id=914-55784&mode=design#751456561